src/test/java/bootstrap/DriverTest.java

```diff
@@ -0,0 +1,108 @@
+package bootstrap;
+
+import domain.Astar;
+import domain.Cell;
+import domain.CellComparator;
+import domain.GenerateMaze;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.*;
+
+public class DriverTest {
+
+    @Test
+    public void ValidPath() {
+        List<Integer> wallsList = new ArrayList<>();
```

Comment:
You can place all repeated code in public void setUp(){} and annotate it with @Before - this way it will get called before each test case.